cmd/gnmi_cli/gnmi_cli.go

```diff
@@ -212,13 +232,36 @@ func executeGet(ctx context.Context) error {
 	if err := proto.UnmarshalText(*reqProto, r); err != nil {
 		return fmt.Errorf("unable to parse gnmi.GetRequest from %q : %v", *reqProto, err)
 	}
-	c, err := gclient.New(ctx, client.Destination{
-		Addrs:       q.Addrs,
-		Target:      q.Target,
-		Timeout:     q.Timeout,
-		Credentials: q.Credentials,
-		TLS:         q.TLS,
-	})
+
+	if strings.Compare(*encodingType, "JSON") == 0 {
+		r.Encoding = gpb.Encoding_JSON
+
+		// For the stratum
+	} else if strings.Compare(*encodingType, "PROTO") == 0 {
+		r.Encoding = gpb.Encoding_PROTO
+	}
```

Comment:
should give an error if it's anything else

Reply:
Done